code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -3467,6 +3467,72 @@
 	glass_name = "glass of Winter Offensive"
 	glass_desc = "Proven to be more successful than the campaign."
 
+// Skrellian drinks
+//====================
+// Some are alocholic, some are not
+
+/datum/reagent/alcohol/ethanol/thirdincident
+	name = "The Third Incident"
+	id = "thirdincident"
+	color = "#1936a0"
+	strength = 10
+	description = "A controversial drink popular with the punk youth of the Jargon Federation. Represents blood, eggs, and tears."
+	taste_description = "genophage sadness"
+
+	glass_icon_state = "thirdincident"
+	glass_name = "glass of the Third Incident"
+	glass_desc = "A controversial drink popular with the punk youth of the Jargon Federation. Represents blood, eggs, and tears."
+
+/datum/reagent/drink/upsidedowncup
+	name = "Upside-Down Cup"
+	id = "upsidedowncup"
+	color = "#B2110A"
+	description = "The classic Skrell meme."
+	taste_description = "esoteric humor"
+
+	glass_icon_state = "upsidedowncup"
+	glass_name = "glass of Upside-Down Cup"
+	glass_desc = "The classic Skrell meme. It's not actually upside down."
```

Comment:
Replace meme here with joke as well, please.

Reply:
See response to your comment above. I have asked the loredev directly, and they indeed _are_ memes.

Reply:
There's a difference between 'joke' and 'meme' - and meme fits best for the use upside down cup has had in the Skrell playerbase, culture, and lore. A joke is something actually funny to most people, a meme is a cultural idea, and the upside down cup is the pinnacle, nostalgic, original meme.

Reply:
The word is just the best one that fits.

Reply:
As Sleepy has stated she doesn't want it changed into "joke", is this still a wording issue?

Reply:
I've done a bit of a work-around that uses neither "joke" nor "meme" in this instance. ICly, Skrell can just explain how prolific it is to confused aliens.